content/300-guides/200-deployment/201-serverless/300-deploy-to-vercel.mdx

```diff
@@ -6,281 +6,47 @@ metaDescription: 'Learn how to deploy a Next.js application based on Prisma Clie
 
 <TopBlock>
 
-This guide takes you through the steps to set up and deploy a serverless application that uses Prisma to [Vercel](https://vercel.com/). The application uses Next.js to provide a serverless REST API, and Prisma Client to handle fetching, creating, and deleting records from a database.
+This guide takes you through the steps to set up and deploy a serverless application that uses Prisma to [Vercel](https://vercel.com/).
 
 Vercel is a cloud platform that hosts static sites and serverless functions. You can integrate a Vercel project with a GitHub repository to allow you to deploy automatically when you make new commits. [Next.js](https://nextjs.org/) is a full-stack React-based serverless framework that integrates seamlessly with Vercel.
 
-The Next.js app has the following components:
-
-- **Backend**: Next.js API routes that serve a REST API and use Prisma Client to handle database operations on a PostgreSQL database.
-- **Frontend**: Next.js page to interact with the API.
+This guide focuses on how to use Prisma with Next.js and Vercel. We used a Next.js app [found here](https://github.com/prisma/deployment-example-vercel), which has four serverless REST endpoints configured as [Next.js API Routes](https://nextjs.org/docs/api-routes/introduction) and a Next.js page that allows you to call the endpoints and display data.
```

Comment:
I think it would be important to keep this guide general and not framework specific to reduce confusion.

```suggestion
We created an [example application](https://github.com/prisma/deployment-example-vercel) using Next.js you can use as a reference when deploying an application using Prisma to Vercel.
```